test/forge/ReallocateWithdrawTest.sol

```diff
@@ -36,28 +38,15 @@ contract ReallocateWithdrawTest is IntegrationTest {
         withdrawn.push(MarketAllocation(allMarkets[0], 0, morpho.supplyShares(allMarkets[0].id(), address(vault))));
         withdrawn.push(MarketAllocation(allMarkets[1], 0, morpho.supplyShares(allMarkets[1].id(), address(vault))));
         withdrawn.push(MarketAllocation(allMarkets[2], 0, morpho.supplyShares(allMarkets[2].id(), address(vault))));
+        supplied.push(MarketAllocation(idleParams, type(uint256).max, 0));
 
         vm.prank(ALLOCATOR);
         vault.reallocate(withdrawn, supplied);
 
         assertEq(morpho.supplyShares(allMarkets[0].id(), address(vault)), 0, "morpho.supplyShares(0)");
         assertEq(morpho.supplyShares(allMarkets[1].id(), address(vault)), 0, "morpho.supplyShares(1)");
         assertEq(morpho.supplyShares(allMarkets[2].id(), address(vault)), 0, "morpho.supplyShares(2)");
-        assertEq(vault.idle(), INITIAL_DEPOSIT, "vault.idle() 1");
-    }
-
-    function testReallocateWithdrawMax() public {
```

Comment:
Why was this test removed ?

Reply:
It's a duplicate now compared to the test above